src/app/templates/test/_karma.conf.js

```diff
@@ -1,34 +1,23 @@
 module.exports = function(config) {
-  var browsers = config.browsers;
-  var frameworks = ['qunit'];
-  var plugins = ['karma-qunit'];
-
-  var addBrowserLauncher = function(browser) {
-    plugins.push('karma-' + browser.toLowerCase() + '-launcher');
+  var detectBrowsers = {
+    enabled: false,
```

Comment:
yeah it my testing it does and it also says so in the [README](https://github.com/litixsoft/karma-detect-browsers)